### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by baseballlover723**
Mostly just formatting, a couple of changes though

### CODE REVIEW

File: src/game/Game.java [hunk 1]
```diff
@@ -133,33 +133,27 @@ public boolean checkCoor(Coordinate coor) {
...
-            this.moves.add(m);
-            endMove();
-            return true;
-        }
-    }
-
-	
-	
-	
+	public boolean move(MoveCommand m) {
```
Comment (baseballlover723) [inline src/game/Game.java:136]: Mostly just formatting

File: src/game/Game.java [hunk 2]
```diff
@@ -224,15 +222,14 @@ private void endMove() {
...
 	 */
 	private void checkWin() {
-		if(this.getPlayerTurn() == 0){
+		if (this.getPlayerTurn() == 0) {
 			return;
 		}
-		
-		Owner lastPlayer = Owner.Player2;
+
+		Owner lastPlayer = this.getPlayerTurn() == 1 ? Owner.Player1 : Owner.Player2;
```
Comment (baseballlover723) [inline src/game/Game.java]: functional change

File: src/game/Game.java [hunk 3]
```diff
@@ -241,31 +238,32 @@ private void checkWin() {
 				}
 			}
 		}
-		
-		Owner otherPlayer =Owner.Player1;
+
+		Owner otherPlayer = this.getPlayerTurn() == 1 ? Owner.Player2 : Owner.Player1;
```
Comment (baseballlover723) [inline src/game/Game.java]: functional change

File: src/game/Game.java [hunk 4]
```diff
@@ -241,31 +238,32 @@ private void checkWin() {
...
 		for (int i = 0; i < 8; i++) {
 			if (this.currentBoard.pieceAt(new Coordinate(i, 7))) {
 				if (this.currentBoard.getPieceAt(new Coordinate(i, 7)).equals(new Rabbit(otherPlayer))) {
-					winner = this.getPlayerTurn();
+					// Mapping from 1->2, 2->1
+					winner = 3 - this.getPlayerTurn();
```
Comment (baseballlover723) [inline src/game/Game.java]: functional chagne

File: src/move_commands/RegularMove.java [hunk 5]
```diff
@@ -42,7 +40,7 @@ public boolean isValidMove() {
 		if (isFrozen(originalPlace)) {
 			return false;
 		}
-		if (this.originalBoard.getPieceAt(newPlace) != null) {
+		if (this.originalBoard.pieceAt(newPlace)) {
```
Comment (baseballlover723) [inline src/move_commands/RegularMove.java:43]: style change

File: test/game/TestCoordinate.java [hunk 6]
```diff
@@ -105,99 +105,96 @@ public void testInvalidIsNeverEqual() {
...
 		Coordinate invalid1 = new Coordinate(-1, -1);
 		Coordinate invalid2 = new Coordinate(8, 8);
-		
+
 		assertNotEquals(valid, invalid1);
 		assertNotEquals(valid, invalid2);
 		assertNotEquals(invalid1, valid);
 		assertNotEquals(invalid2, valid);
-		
-		assertNotEquals(invalid1, invalid1);
```
Comment (baseballlover723) [inline test/game/TestCoordinate.java:114]: these asserts are no longer valid

**Comment by TSYMYomi** [comment]
Nothing crazy here, seems fine.

File: test/testing/AllTests.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
+package testing;
```
Comment (baseballlover723) [inline test/testing/AllTests.java:1]: this got moved to src/test_runner/allTests.java

**Pull request merged**